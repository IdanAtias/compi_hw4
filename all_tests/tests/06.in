C x;
F y = 8 F;
if(x==y)
{
	F z;
	F w;
}
bool a=1 F;

