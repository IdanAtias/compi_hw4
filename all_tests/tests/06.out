-- end scope --
var:z:2:FAHRENHEIT
var:w:3:FAHRENHEIT
line 8: type mismatch
